config/config_old.py

```diff
@@ -27,7 +27,7 @@ class Development(Base):
     """ development config """
 
     DEBUG = True
-    SQLALCHEMY_DATABASE_URI = "postgresql://postgres:postgres@database/cranecloud"
+    SQLALCHEMY_DATABASE_URI = "postgresql://postgres:postgres@database/cranecloud2"
```

Comment:
revert the database name back to cranecloud. config_old.py is no longer being used.